PaddleCV/object_detection/configs/mask_rcnn_r50_fpn_1x.yml

```diff
@@ -116,34 +115,34 @@ OptimizerBuilder:
 
 MaskRCNNTrainFeed:
   batch_size: 1
-  batch_transforms:
-  - !PadBatch
-    pad_to_stride: 32
   dataset:
     dataset_dir: dataset/coco
     annotation: annotations/instances_train2017.json
     image_dir: train2017
+  batch_transforms:
+  - !PadBatch
+    pad_to_stride: 32
   num_workers: 2
-  use_padded_im_info: False
+  use_padded_im_info: false
```

Comment:
we can remove this flag in config? @jerrywgz

Reply:
Sure, it can be removed

Reply:
done